home/views.py

```
@@ -1,12 +1,12 @@
 from django.shortcuts import render
 from django.views.decorators.http import require_http_methods
 
-from courses.models import Courses, Enrollment, CourseTeacher
+from courses.models import Course, Enrollment, CourseTeacher
 
 
 @require_http_methods(["GET"])
 def home(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `home`

(ANN201)